set.mm

```diff
@@ -734583,9 +734582,8 @@ Basic algebraic structures (extension)
     cbvmpt2x2.7 $e |- ( y = z -> A = D ) $.
     cbvmpt2x2.8 $e |- ( ( y = z /\ x = w ) -> C = E ) $.
     $( Rule to change the bound variable in a maps-to function, using implicit
-       substitution.  This version of ~ cbvmpt2 allows ` A ` to be a function
-       of ` y ` , analogous to ~ cbvmpt2x .  (Contributed by AV,
-       30-Mar-2019.) $)
+       substitution.  This version of ~ cbvmpo allows ` A ` to be a function of
+       ` y ` , analogous to ~ cbvmpox .  (Contributed by AV, 30-Mar-2019.) $)
     cbvmpt2x2 $p |- ( x e. A , y e. B |-> C ) = ( w e. D , z e. B |-> E ) $=
```

Comment:
Are the names of concerbed theorems in mathboxes also changed automatically, or does the maxbox owners have to take care of it?

Reply:
As long as the reference is in set.mm in this repository I have been changing them, whether mathbox or not (the automated tests require this and it just seems like a good idea).

If people have their own branches or forks or copies outside git, they will need to update them.